client/src/pages/guide/english/linux/basic-linux-commands/index.md

```diff
@@ -39,3 +39,6 @@ When starting out with linux, there are some basic commands everyone should know
 
  11. **less** - view file content  
 - Use `less filename.txt` to view contents of a file and navigate through them. By default, less will go through the file page by page.
+
+ 12. **kill** - kills the process   
+- Used to kill any suspended or ongoing process or process group using the PID number.
```

Comment:
Write PID in full, so people don't have to look it up separately.

And technically it's not just about killing, you can send various useful signals.